lib/web/terminal.go

```diff
@@ -225,7 +219,37 @@ func (t *TerminalHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	ws.SetReadDeadline(deadlineForInterval(t.params.KeepAliveInterval))
+	ws.SetReadDeadline(deadlineForInterval(t.keepAliveInterval))
```

Comment:
This breaks the ability to set `keepAliveInterval` in the request, which we use in tests via `withKeepaliveInterval`.

Reply:
`keepAliveInterval` has been added back in 👍